UnitsNet/GeneratedCode/Quantities/VolumeFlow.NetFramework.g.cs

```diff
@@ -65,6 +117,32 @@ public VolumeFlow(double numericValue, VolumeFlowUnit unit)
             _unit = unit;
         }
 
+        /// <summary>
+        /// Creates an instance of the quantity with the given numeric value in units compatible with the given <see cref="UnitSystem"/>.
+        /// </summary>
+        /// <param name="numericValue">The numeric value  to contruct this quantity with.</param>
+        /// <param name="unitSystem">The unit system to create the quantity with.</param>
+        public VolumeFlow(double numericValue, UnitSystem unitSystem)
+        {
+            if(unitSystem == null) throw new ArgumentNullException(nameof(unitSystem));
+
+            _value = Guard.EnsureValidNumber(numericValue, nameof(numericValue));
+            _unit = GetUnitFor(unitSystem.BaseUnits);
+        }
+
+        /// <summary>
+        /// Creates an instance of the quantity with the given numeric value in units compatible with the given <see cref="BaseUnits"/>.
+        /// </summary>
+        /// <param name="numericValue">The numeric value  to contruct this quantity with.</param>
+        /// <param name="baseUnits">The base units to create the quantity with.</param>
+        public VolumeFlow(double numericValue, BaseUnits baseUnits)
```

Comment:
I think maybe you missed my last comment, so I'll add it back below. I think we should choose only one of the two ctors. I prefer `UnitSystem` over `BaseUnits`, because I think it's easier to relate to. Base units is in my mind a bit abstract/unfamiliar.

> It seems a bit odd to me that we can pass in both UnitSystem and BaseUnits to the ctor here.
Also, somewhat out of scope of PR, but what value does UnitSystem bring beyond BaseUnits? Do we expect it to have anything more than a list of base units at some point? Could we merge the two?

Reply:
See my above statement about UnitSystem vs BaseUnits. However I'll remove the base units constructor and just keep UnitSystem.